### PR TITLE
Add diff method to DateUtil

### PR DESCRIPTION
**Pull request by SegmentationFaultEnjoyer**
Added diff method to DateUtil

### CODE REVIEW

File: packages/utils/src/date.ts [hunk 1]
```diff
@@ -107,4 +107,17 @@ export class DateUtil {
   ): boolean {
     return this._dayjs(targetDate).isSameOrBefore(comparisonDate)
   }
+
+  public static diffs(
```
Comment (napalmpapalam) [inline packages/utils/src/date.ts]: why diff**s**?

File: packages/utils/src/date.ts [hunk 2]
```diff
@@ -107,4 +107,17 @@ export class DateUtil {
...
+    targetDate: ConfigType,
+    comparisonDate: ConfigType,
+    unit?: UnitType,
+    float?: boolean,
```
Comment (napalmpapalam) [inline packages/utils/src/date.ts]: what does float mean? 

```ts
isTruncated: boolean = false
```

File: packages/utils/src/date.ts [hunk 3]
```diff
@@ -107,4 +107,17 @@ export class DateUtil {
   ): boolean {
```
Comment (napalmpapalam) [inline packages/utils/src/date.ts:107]: 1. Add change log to the README.md with incremented version, according to semantic versioning
2. Change version in package.json

**Review by napalmpapalam — CHANGES_REQUESTED** [review]

File: packages/utils/src/date.ts (hunk 1 above)
Comment (SegmentationFaultEnjoyer) [reply packages/utils/src/date.ts]: changed to diff

File: packages/utils/src/date.ts (hunk 2 above)
Comment (SegmentationFaultEnjoyer) [reply packages/utils/src/date.ts]: that naming was taken from original diff method from dayjs. Now renamed to what you suggested

File: packages/utils/src/date.ts (hunk 3 above)
Comment (SegmentationFaultEnjoyer) [reply packages/utils/src/date.ts:107]: done

**Review by napalmpapalam — APPROVED** [review]

**Pull request merged**